### PR TITLE
Fix minor issues in the instructions

### PR DESCRIPTION
**Pull request by pudelkoM**
- Add hint to leave mininet running after exercise 1
 - Add exercise 1 solution in exercise 2
 - Add text about missing links in exercise 3
 - Fix MAC address in L2BridgingComponent.java

### CODE REVIEW

File: EXERCISE-2.md [hunk 1]
```diff
@@ -230,6 +230,27 @@ Starting onos    ... done
 You will need to go back to Exercise 1 and install forwarding rules to
 re-establish pings between `h1a` and `h1b` for later parts of this exercise.
 
+If you could not complete Exercise 1, you can use the following P4Runtime-sh
+commands to enable connectivity:
+
+<details><summary>Solution Exercise 1</summary>
```
Comment (ccascone) [inline EXERCISE-2.md]: We don't use html tags anywhere else in the instructions (besides for images). The reason is that we want instructions to be consumable from an IDE or terminal, i.e. not rendered on GitHub. It's easier to read without HTML tags.

File: EXERCISE-3.md [hunk 2]
```diff
@@ -306,7 +306,8 @@ When asked, use the username `onos` and password
 `rocks`.
 
 You should wee 4 devices in the topology view, corresponding to the 4 switches
```
Comment (ccascone) [inline EXERCISE-3.md]:
```suggestion
You should see 4 devices in the topology view, corresponding to the 4 switches
```

File: Makefile [hunk 3]
```diff
@@ -107,7 +107,7 @@ p4-build: p4src/main.p4
 		p4src/main.p4
 	@echo "*** P4 program compiled successfully! Output files are in p4src/build"
 
-p4-test:
+p4-test: _docker_build
```
Comment (ccascone) [inline Makefile]: Not required. `_docker_build` is called by `make deps`

Comment (pudelkoM) [reply Makefile]: But `make deps` is never used elsewhere and also not mentioned in the exercises.

File: EXERCISE-2.md (hunk 1 above)
Comment (pudelkoM) [reply EXERCISE-2.md]: Got it. Will revert to plain markdown.

File: Makefile (hunk 3 above)
Comment (ccascone) [reply Makefile]: It is here: https://github.com/opennetworkinglab/ngsdn-tutorial/tree/princeton19#download--upgrade-dependencies

File: app/src/main/java/org/onosproject/ngsdn/tutorial/L2BridgingComponent.java [hunk 4]
```diff
@@ -210,7 +210,7 @@ private void insertMulticastFlowRules(DeviceId deviceId) {
         // Modify P4Runtime entity names to match content of P4Info file (look
         // for the fully qualified name of tables, match fields, and actions.
         // ---- START SOLUTION ----
-        // Match ARP request - Match exactly FF:FF:FF:FF:FF
+        // Match ARP request - Match exactly FF:FF:FF:FF:FF:FF
```
Comment (ccascone) [inline app/src/main/java/org/onosproject/ngsdn/tutorial/L2BridgingComponent.java]: Can you please update the corresponding solution file?

Comment (pudelkoM) [reply app/src/main/java/org/onosproject/ngsdn/tutorial/L2BridgingComponent.java]: Solution code was OK, just the text

File: EXERCISE-3.md [hunk 5]
```diff
@@ -305,8 +305,9 @@ bar. If you are using the tutorial VM, open up a browser (e.g. Firefox) to
 When asked, use the username `onos` and password
 `rocks`.
 
-You should wee 4 devices in the topology view, corresponding to the 4 switches
-of our 2x2 fabric. Press `L` to show device labels.
+You should see 4 devices in the topology view, corresponding to the 4 switches
+of our 2x2 fabric. Press `L` to show device labels. Because we disabled link
+discovery, the ONOS UI will not show any links between the devices.
```
Comment (pudelkoM) [inline EXERCISE-3.md]: Reword: "not yet implemeted"

File: Makefile (hunk 3 above)
Comment (pudelkoM) [reply Makefile]: Remove target, should be done as part of provisioning

**Pull request merged**